__tests__/app.test.js

```diff
@@ -106,7 +106,49 @@ describe ('GET /api/articles/:article_id',() => {
                 expect(article).toHaveProperty("votes", expect.any(Number));   
                 expect(article).toHaveProperty("article_img_url", expect.any(String));
                 expect(article).toHaveProperty("comment_count", expect.any(Number));
+                expect(article).not.toHaveProperty("body"); 
             })        
         })
       }); 
+  })
+
+  //Task 6 GET /api/articles/:article_id/comments
+describe ('GET /api/articles/:article_id/comments',() => {
+    test ('200:should return comments by article_id',() => {
+    return request(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then (({body}) => {
+        const {comments} = body;
+        expect(comments).toBeSortedBy("created_at",{descending: true,});
+        comments.forEach((comment) => {
```

Comment:
please include a length check to avoid a false positive test if the array we're iterating over is empty

Reply:
Added a length check